<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EditorResources.qrc">
    <normaloff>:/images/s_icon.png</normaloff>:/images/s_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">body {
	background: black;
}

QDialogButtonBox  QPushButton
{

}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>281</width>
      <height>621</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>279</width>
       <height>619</height>
      </rect>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Контраст чёрно-белый</string>
      </property>
      <widget class="QSpinBox" name="contrastMin_spinBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="contrastMax_spinBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max:</string>
       </property>
      </widget>
      <widget class="QToolButton" name="contrastOk_btn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>✔</string>
       </property>
       <property name="icon">
        <iconset resource="EditorResources.qrc">
         <normaloff>:/images/icon-save.png</normaloff>:/images/icon-save.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Яркость</string>
      </property>
      <widget class="QSlider" name="brightnessSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-255</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="brightnessOk_btn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="EditorResources.qrc">
         <normaloff>:/images/icon-save.png</normaloff>:/images/icon-save.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Отображение</string>
      </property>
      <widget class="QComboBox" name="viewTypeComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
      <widget class="QToolButton" name="viewType_btn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>✔</string>
       </property>
       <property name="icon">
        <iconset resource="EditorResources.qrc">
         <normaloff>:/images/icon-save.png</normaloff>:/images/icon-save.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Контраст цветной</string>
      </property>
      <widget class="QToolButton" name="contrastrgbOk_btn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>✔</string>
       </property>
       <property name="icon">
        <iconset resource="EditorResources.qrc">
         <normaloff>:/images/icon-save.png</normaloff>:/images/icon-save.png</iconset>
       </property>
      </widget>
      <widget class="QSlider" name="contrastrgb_Slider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-255</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="discard_pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>580</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>discard</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Бинаризация</string>
      </property>
      <widget class="QToolButton" name="binarOk_btn">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>✔</string>
       </property>
       <property name="icon">
        <iconset resource="EditorResources.qrc">
         <normaloff>:/images/icon-save.png</normaloff>:/images/icon-save.png</iconset>
       </property>
      </widget>
      <widget class="QSlider" name="binarTreshold_Slider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>127</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="rgbStatus_Label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>640</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rgb(255, 150, 10)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>640</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*click mouse for tracking</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="imagesTabs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>851</width>
      <height>631</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Image 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>591</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="commonSliderLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>640</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>640</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Slider value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xPos_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPos_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Изоображение</string>
    </property>
    <addaction name="actionInfoImage"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Операции</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Поворот изоображения</string>
     </property>
     <addaction name="action_TransformAngleLeft"/>
     <addaction name="action_TransformAngleRight"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Изменить расположение</string>
     </property>
     <addaction name="action_TransformByHorizontal"/>
     <addaction name="action_TransformByVertical"/>
    </widget>
    <addaction name="action_symmetryVerticalTransform"/>
    <addaction name="action_symmetryHorizontalTransform"/>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actionInfoImage">
   <property name="text">
    <string>Информация об изоображении</string>
   </property>
  </action>
  <action name="action_symmetryVerticalTransform">
   <property name="text">
    <string>Симметричное вертикальное отображение</string>
   </property>
  </action>
  <action name="action_symmetryHorizontalTransform">
   <property name="text">
    <string>Симметричное горизонтальное отображение</string>
   </property>
  </action>
  <action name="action_TransformAngleLeft">
   <property name="text">
    <string>Поворот влево</string>
   </property>
  </action>
  <action name="action_TransformAngleRight">
   <property name="text">
    <string>Поворот вправо</string>
   </property>
  </action>
  <action name="action_TransformByHorizontal">
   <property name="text">
    <string>По горизонтали</string>
   </property>
  </action>
  <action name="action_TransformByVertical">
   <property name="text">
    <string>По вертикали</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Новый файл</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="EditorResources.qrc"/>
 </resources>
 <connections/>
</ui>
